jlm/llvm/ir/operators/Phi.cpp

```diff
@@ -168,11 +168,24 @@ rvoutput::~rvoutput()
 rvargument::~rvargument()
 {}
 
+rvargument &
+rvargument::CopyTo(rvsdg::region & region, rvsdg::structural_input *)
```

Comment:
I assume it is intentional to ignore the `structural_input`, as recursion variables do not take inputs.
Would it improve clarity if we added an assert to ensure that the input is in fact `nullptr`?

Reply:
I have added `JLM_ASSERT()`s. The only concern I have is that this might result in unused variables errors during compilation when we compile in release without asserts. I have not tested.

Reply:
I also opened issue #579 to not forget that we should do something about this.

Reply:
It compiles, I just checked. I think what saved us is the `-Wno-unused-parameter` compilation flag. We need it in order to convince the compiler to compile with the LLVM headers.